---
layout: archive
title: "CV"
permalink: /cv/
author_profile: true
redirect_from:
  - /resume
---

{% include base_path %}

Education
======
* Ph.D in Bioinformatics and RNA biology, Univeristy of Regensburg, 2022
* M.S.c in Genomics and Systems Biology, University of Gothenburg, 2016
* Erasmus exchange, University of Aberdeen, 2013
* B.S.c in Biotechnology, University of Ljubljana, 2014

Work experience
======
* 2025 - ongoing: Computational Scientist
  * PI: Dr. Sabine Taschner-Mandl

* 2022 - 2025: Postdoc
  * Epigenetics
  * Tumor microenvironment
  * Neuroblastoma
  * scRNA-seq | snATAC-seq
  * PI(s): Dr. Sabine Taschner-Mandl and Dr. Florian Halbritter
 
* 2018 - 2022: PhD student
  * Unstable nucleosomes
  * RNA:DNA triplexes
  * PI: prof. Dr. Gernot Längst
 
* 2016 - 2018: Bioinformatician
  * Optimization of DamID-seq pipeline
  * Optimization of DamID protocol for use in mammalian cells
  * Received AG FOND Research grant: Alternative modeling of neuroblastoma progress using patient derived iPSCs.
  * PI: prof. Dr. Chandrasekhar Kanduri
 
* 2015 - 2016: MSc project
  * DNA hydroxymethylation in CLL patients
  * CASC15 lncRNA in neuroblastoma 
  * PI: prof. Dr. Chandrasekhar Kanduri and Dr. Tanmoy Mondal
 
* January – July 2012: Student researcher
  * Characterization of mycobacteria proteins
  * Supervisor(s): Dr. Marjetka Podobnik and prof. Dr. Gregor Anderluh
  
Skills
======
* Coding [R, Python, bash, SQL]
* Mathematical modelling
* NGS analysis [ChIP-seq, RNA-seq, smallRNA-seq analysis, MNase-seq, ATAC-seq, DamID-seq, hMeDIp/MeDIP-seq, BS-seq]
* Single cell analysis [scRNA-seq, scATAC-seq]
* NGS library construction
* Gene perturbations [CRISPR/Cas9, sh/siRNA, LNA, OE]
* Cell culture [primary patient cell lines, hESCs, iPSCs]
* Biochemical assays [Microscale thermophoresis, western blot]


Publications
======
[1] Wernig-Zorc, S., Kugler, F., Schmutterer, L., Räß, P., Hausmann, C., Holzinger, S., Längst, G., & Schwartz, U. (2024). nucMACC: An MNase-seq pipeline to identify structurally altered nucleosomes in the genome. Science Advances, 10(27), 9740. https://doi.org/10.1126/SCIADV.ADM9740

[2] Saldana-Guerrero, I. M., Montano-Gutierrez, L. F., Boswell, K., Hafemeister, C., Poon, E., Shaw, L. E., Stavish, D., Lea, R. A., Wernig-Zorc, S., Bozsaky, E., Fetahu, I. S., Zoescher, P., Pötschger, U., Bernkopf, M., Wenninger-Weinzierl, A., Sturtzel, C., Souilhol, C., Tarelli, S., Shoeb, M. R., Bozatzi, P., Rados, M., Guarini, M., Buri, M. C., Weninger, W., Putz, E. M., Huang, M., Ladenstein, R., Andrews, P. W., Barbaric, I., Cresswell, G. D., Bryant, H. E., Distel, M., Chesler, L., Taschner-Mandl, S., Farlik, M., Tsakiridis, A., & Halbritter, F. (2024). A human neural crest model reveals the developmental impact of neuroblastoma-associated chromosomal aberrations. Nature Communications, 15(1), 1–25. https://doi.org/10.1038/s41467-024-47945-7

[3] Wernig-Zorc, S., Schwartz, U., Martínez, P., Inalef, J., Pavicic, F., Ehrenfeld, P., Längst, G., & Maldonado Affiliations, R. (2024). The long non-coding RNA MALAT1 modulates NR4A1 expression through a downstream regulatory element in specific cancer-cell-types. International Journal of Molecular Sciences, 25, 5515. https://doi.org/10.1101/2023.03.09.531856

[4] Fitzgerald, E., Sinton, M. C., Wernig-Zorc, S., Morton, N. M., Holmes, M. C., Boardman, J. P., & Drake, A. J. (2021). Altered hypothalamic DNA methylation and stress-induced hyperactivity following early life stress. Epigenetics & chromatin, 14(1), 14–31. https://doi.org/10.1186/S13072-021-00405-8

[5] Sinton, M. C., Meseguer-Ripolles, J., Lucendo-Villarin, B., Wernig-Zorc, S., Thomson, J. P., Carter, R. N., Lyall, M. J., Walker, P. D., Thakker, A., Meehan, R. R., Lavery, G. G., Morton, N. M., Ludwig, C., Tennant, D. A., Hay, D. C., & Drake, A. J. (2021). A human pluripotent stem cell model for the analysis of metabolic dysfunction in hepatic steatosis. iScience, 24(1), 101931. https://doi.org/10.1016/j.isci.2020.101931

[6] Wernig-Zorc, S., Yadav, M. P., Kopparapu, P. K., Bemark, M., Kristjansdottir, H. L., Andersson, P. O., Kanduri, C., & Kanduri, M. (2019). Global distribution of DNA hydroxymethylation and DNA methylation in chronic lymphocytic leukemia. Epigenetics and Chromatin, 12(4). https://doi.org/10.1186/s13072-018-0252-7

[7] Mondal, T., Juvvuna, P. K., Kirkeby, A., Mitra, S., Kosalai, S. T., Traxler, L., Hertwig, F., Wernig-Zorc, S., Miranda, C., Deland, L., Volland, R., Bartenhagen, C., Bartsch, D., Bandaru, S., Engesser, A., Subhash, S., Martinsson, T., Carén, H., Akyürek, L. M., Kurian, L., Kanduri, M., Huarte, M., Kogner, P., Fischer, M., & Kanduri, C. (2018). Sense-Antisense lncRNA Pair Encoded by Locus 6p22.3 Determines Neuroblastoma Susceptibility via the USP36-CHD7-SOX9 Regulatory Axis. Cancer Cell, 33(3), 417-434.e7. https://doi.org/10.1016/j.ccell.2018.01.020
